Assets/Tests/InputSystem/CoreTests_API_Touch.cs

```diff
@@ -0,0 +1,138 @@
+using System.Collections;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.InputSystem;
+using UnityEngine.InputSystem.LowLevel;
+using UnityEngine.TestTools;
+
+partial class CoreTests
+{
+    private static bool TouchesAreEqual(Touch first, Touch second)
+    {
+        return
+            first.fingerId == second.fingerId &&
+            first.position == second.position &&
+            first.rawPosition == second.rawPosition &&
+            first.deltaPosition == second.deltaPosition &&
+            first.deltaTime == second.deltaTime &&
+            first.tapCount == second.tapCount &&
+            first.phase == second.phase &&
+
+            first.pressure == second.pressure &&
+            first.maximumPossiblePressure == second.maximumPossiblePressure &&
+            first.type == second.type &&
+            first.altitudeAngle == second.altitudeAngle &&
+            first.azimuthAngle == second.azimuthAngle &&
+            first.radius == second.radius &&
+            first.radiusVariance == second.radius;
+    }
+
+    [UnityTest]
+    [Category("API")]
+    public IEnumerator API_CanReadTouchEventsThroughTouchesAPI()
+    {
+        Assert.That(Input.touchSupported, Is.False);
```

Comment:
Prob good to try the other related APIs here before Touchscreen is added to make sure we get the expected defaults.

Reply:
Also, test out of range access with `GetTouch`.